README.md

```diff
@@ -238,6 +238,21 @@ docker run --rm -v /tmp:/tmp -p 9090:9090 -v $(pwd):/app:rw -t ghcr.io/cyclonedx
 
 Use curl or your favorite tool to pass arguments to the `/sbom` route.
 
+### Server arguments
```

Comment:
Nice idea!

We also have a dedicated docsify folder. You can add a page there and link it to the sidebar. And finally, running prettier on the markdown files will clean up the tables.

https://github.com/CycloneDX/cdxgen/tree/master/docs

Reply:
Thanks for the feedback, I actually hadn't seen the https://cyclonedx.github.io/cdxgen/#/ until you mentioned this 😄